docs/pom.xml

```diff
@@ -6,7 +6,7 @@
     <groupId>io.quarkiverse.openapi.generator</groupId>
     <artifactId>quarkus-openapi-generator-client-parent</artifactId>
     <version>3.0.0-SNAPSHOT</version>
-    <relativePath>../pom.xml</relativePath>
+    <relativePath>../client/pom.xml</relativePath>
```

Comment:
This should be `../pom.xml` and the parent artifactId `quarkus-openapi-generator-parent`